src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -15,428 +15,108 @@ redirects:
-
-OpenTelemetry is still an emerging standard, so your choices may be affected by what's available. You can check on the current state of the specification at the [OpenTelemetry site](https://opentelemetry.io).
-
-The current state of language-specific OpenTelemetry APIs and SDKs varies: some languages are still pre-alpha and may be missing instructions on how to instrument your service. Most languages have some implementation of traces that is sufficient to start exporting data to New Relic. Check out this [table in GitHub](https://github.com/open-telemetry/opentelemetry-specification/blob/master/spec-compliance-matrix.md) that provides an overview of the state of OpenTelemetry specification compliance for each language.
-
-For languages where New Relic does not currently provide an agent or SDK, OpenTelemetry may offer you a good alternative. Also, in cases where you want explicit control over sampling of your telemetry data, OpenTelemetry provides a lot of flexibility.
-
-As OpenTelemetry matures, we will continue to support new OpenTelemetry data models and provide you curated UI experiences in our platform.
-
-### New Relic agents [#apm-agents]
-
-In general, New Relic's <InlinePopover type="apm" /> and infrastructure agents will collect more telemetry data for your services and hosts. Plus, they offer a wide range of configuration options and an extensive set of auto-instrumentation capabilities.
-
-New Relic's APM agents offer detailed transaction trace visibility for individual services. They also offer predefined sampling to balance the performance impact of your instrumentation against the need to capture enough data to gain helpful insights.
-
-<Callout variant="important">
-  <DoNotTranslate>**Running New Relic agents and OpenTelemetry tooling in the same process**</DoNotTranslate>
-
-  New Relic agents and OpenTelemetry tooling (SDKs, agents, etc.) are among
-  many APM products on the market.
-
-  For a given language, nearly all APM products operate by using the same
-  low-level language and runtime hooks available–most of which alter the
-  behavior of your code at run time. Depending on your language's architecture,
-  this is often achieved via techniques like byte-code manipulation or monkey
-  patching.
-
-  Due to the complex ways in which APM products alter your running code,
-  there can be no guarantees that one product will be compatible with another
-  product running in the same process. At best, they’ll live side by side
-  unbeknownst to the other and independently generate telemetry from your
-  application. At worst, they’ll stomp on each other leading to unpredictable
-  behavior.
-
-  New Relic cannot make any assurances that our APM agents are compatible with
-  a different APM product running in the same process. We recommend you pick the
-  one–and only one–that best fits your needs.
-</Callout>
-
-## How OpenTelemetry works with New Relic [#how-it-works]
```

Comment:
This content is superseded by "OpenTelemetry -> OpenTelemetry data in New Relic"